schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -40,6 +40,8 @@
                             },
                             {
                                 "oneOf": [
+                                    { "$ref": "http://schema.management.azure.com/schemas/2015-05-21-preview/Microsoft.DevTestLabs.json#/definitions/labs" },
+                                    { "$ref": "http://schema.management.azure.com/schemas/2015-05-21-preview/Microsoft.DevTestLabs.json#/definitions/environments" },
```

Comment:
Please add these references to schemas/2015-01-01/deploymentTemplate.json as well.